src/registry_schemas/example_data/schema_data.py

```diff
@@ -25,7 +25,9 @@
             'certifiedBy': 'full name',
             'email': 'no_one@never.get',
             'filingId': 1,
-            'routingSlipNumber': '123456789'
+            'routingSlipNumber': '123456789',
+            'waveFees': False,
```

Comment:
Typo: should be "waiveFees".

Please do global search & replace.

Reply:
I concur.

Reply:
done